TABSwiftLayout/Constraint.swift

```diff
@@ -251,11 +262,6 @@ public struct Constraint: ConstraintDefinition {
     self.priority = 250
```

Comment:
I don't see why not, there is not that many, and maybe only the constants being used could be mapped (i.e. "Low" and "Required")